packages/blade/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -123,7 +123,36 @@ const ProgressBar = ({
   const hasLabel = label && label.trim()?.length > 0;
   const isMeter = variant === 'meter';
   const progressValue = clamp(value, min, max);
-  const percentageProgressValue = ((progressValue - min) * 100) / (max - min);
+  const percentageProgressValue = Math.floor(((progressValue - min) * 100) / (max - min));
+  const shouldShowPercentage = showPercentage && !isMeter && !isIndeterminate;
+  const accessibilityProps: {
```

Comment:
Let pick from the AccessibilityProps if possible: 

```ts
Pick<AccessibilityProps, 'label' | 'valueMax' | 'valueNow' | 'valueMin' | 'valueText'>;
```

Reply:
Makes sense